.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: "🚀 Feature request"
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+projects: ''
+assignees: ''
+---
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context about the feature request here.
```

Comment:
Maybe clearly specify in "Additional context" in what cases that would be useful.